src/Todo.test.tsx

```diff
@@ -1,21 +1,32 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import {fireEvent, render, screen} from '@testing-library/react';
 import Todo from './Todo';
+import {mockAddTodo, mockFetchTodoList} from "./mocks/mock";
 
-test('Has a submit button', () => {
-  render(<Todo/>);
-  const submitButton = screen.getByText('Add');
-  expect(submitButton).toBeInTheDocument();
+test('Has a submit button',   () => {
+    render(<Todo fetchList={mockFetchTodoList} addTodo={mockAddTodo}/>)
+
+    const submitButton = screen.getByTestId('add-button');
+
+    expect(submitButton).toBeInTheDocument();
 });
 
-test('Has an h1 header that says Todo List', () => {
-  render(<Todo/>);
+test('Has an h1 header that says Todo List',  () => {
+  render(<Todo fetchList={mockFetchTodoList} addTodo={mockAddTodo}/>);
+
   const header = screen.getByText('ToDo List:');
+
   expect(header).toBeInTheDocument();
 })
 
-test('Has an ul', async () => {
-  render(<Todo/>);
-  const list = screen.getByRole('list')
-  expect(list).toBeInTheDocument();
-})
+test('Task is added when button is pressed',  () => {
+  render(<Todo fetchList={mockFetchTodoList} addTodo={mockAddTodo}/>)
+
+  const input = screen.getByTestId('add-task')
+  fireEvent.change(input, { target: { value: "a new task" } })
```

Comment:
Are you sure you need to fire a change event to set this data? Can't you just do something like input.value?

Reply:
Pretty sure. I wasn't able to set the input value without `fireEvent`.